src/Test.UnitTests.Sarif.Multitool.Library/TestData/RewriteCommand/minimal.sarif

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "CodeScanner"
+        }
+      },
+      "results": []
+    }
+  ]
+}
```

Comment:
First, let's not introduce a file on disk if we aren't actually testing against it (as we don't appear to be doing in this case). Second, let's not introduce additional files that are exact duplicates of another checked in file. I'm pretty sure we have a definitive minimal sarif elsewhere. I will sometimes build test helpers that retrieve well-known shared test assets of this kind (rather than depending on checking in a dupe to a directory location associated specifically with the suite under test).